### ISSUE TITLE
Can't load from Stackblitz into bolt, access previous chats or login into bOLT.NEW

### ISSUE THREAD

**Issue by keefcreative** [stackblitz/bolt.new#1995]
### Describe the bug

Asked bolt to complete connection to Firebase, it started and then displayed
<img width="448" alt="Screenshot 2024-11-11 at 14 18 57" src="https://github.com/user-attachments/assets/ae1a60ef-c521-4d00-9fd3-9b263a103ba4">
I can not log into bolt now

This was the error that started everything
<img width="354" alt="image" src="https://github.com/user-attachments/assets/ff10b591-4a2e-4401-9195-68f27e00cb5f">

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-ufdmpl

### Steps to reproduce

Login into Stackblitz,

### Expected behavior

Load previous chat and continue to work on app

### Platform

```
Browser name = Chrome
Full version = 128.0.0.0
Major version = 128
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/128.0.0.0 Safari/537.36
performance.memory = {
  "totalJSHeapSize": 62857956,
  "usedJSHeapSize": 54195324,
  "jsHeapSizeLimit": 4294705152
}
Username = keefcreative
Chat ID = e632e554a636
```

**Comment by endocytosis** [comment]
Hi @keefcreative, 

Appreciate the feedback! Are you able to find your project by going to stackblitz.com, clicking Collections on the left-hand side, and then clicking Bolt Collection. If so, are you able to open that project in StackBlitz and then open in Bolt?

**Comment by keefcreative** [comment]
Hi. Brian,Yeah, I can open in stackblitz, still get the errors but I can’t open it in bolt.I did download a version before I tried to do the fire base implementation. Could I load that into a new bolt chat.I’d lose a fair bit but not everythingSent from my iPhoneOn 11 Nov 2024, at 16:44, Brian ***@***.***> wrote:﻿
Hi @keefcreative,
Appreciate the feedback! Are you able to find your project by going to stackblitz.com, clicking Collections on the left-hand side, and then clicking Bolt Collection. If so, are you able to open that project in StackBlitz and then open in Bolt?
